getting-started.md

```diff
@@ -12,41 +12,64 @@
 
 **Step 2** - Deploy the vCenter Event Broker Appliance OVA to your vCenter Server using the vSphere HTML5 Client. As part of the deployment you will be prompted to provide the following input:
 
-*Networking*
+*Networking* (**Required**)
 
   * Hostname - The FQDN of the vCenter Event Broker Appliance. If you do not have DNS in your environment, make sure the hostname provide is resolvable from your desktop which may require you to manually add a hosts entry. Proper DNS resolution is recommended
   * IP Address - The IP Address of the vCenter Event Broker Appliance
-  * Netmask Prefix - CIDR Notation (e.g. 24 = 255.255.255.0)
+  * Network Prefix - Network CIDR Selection (e.g. 24 = 255.255.255.0)
   * Gateway - The Network Gateway address
   * DNS - DNS Server(s) that will be able to resolve to external sites such as Github for initial configuration. If you have multiple DNS Servers, input needs to be space separated.
```

Comment:
```suggestion
  * DNS - DNS Server(s) that will be able to resolve to external sites such as Github for initial configuration. If you have multiple DNS Servers, input needs to be **space separated**.
```

Reply:
Would it make sense to highlight space separation since many of us are used to "," separation syntax?

Reply:
Yup, make sense.